pyproject.toml

```diff
@@ -43,7 +43,7 @@ setuptools = "65.6.3"
 skops = {version = "0.5.0"}
 xgboost = "1.6.2"
 skorch = "0.11.0"
-torch = "2.3.1"
+torch = "2.2.2"
```

Comment:
I see no way to say "Mac intel" but they are ways to say "Mac"

```
foo = [
    { platform = "darwin", url = "https://example.com/foo-1.0.0-py3-none-macosx_11_0_arm64.whl" },
    { platform = "linux", version = "^1.0", source = "pypi" },
]
```

Do we really want to have versions which differ between OSs, however?